python/services/github_service.py

```diff
@@ -80,15 +81,15 @@ def __get_repos_to_consider_for_archiving(self, repository_type: str) -> list[Re
 
     def __is_repo_ready_for_archiving(self, repository, last_active_cutoff_date, allow_list: list[str]) -> bool:
         latest_commit_position = 0
-        commit = None
+        commit: Commit = None
```

Comment:
The `Commit` Object contains a `commit` attribute of type `GitCommit`

Reply:
Thought I'd make the typing explicit here since writing `commit.commit` is the thing that confused me initially 😂